src/personio_py/client.py

```diff
@@ -341,23 +341,63 @@ def get_absences(self, employees: Union[int, List[int], Employee, List[Employee]
         return self._get_employee_metadata(
             'company/time-offs', Absence, employees, start_date, end_date)
 
-    def get_absence(self, absence_id: int) -> Absence:
+    def get_absence(self, absence: int or Absence, remote_query_id=False) -> Absence:
```

Comment:
`int or Absence` is not a valid type hint, please use `Union[int, Absence]` instead